code/src/containers/forms/product-form/ProductForm.scss

```diff
@@ -0,0 +1,180 @@
+@import "@design-system";
+
+$gridGap: spacing(5);
+$gridGapHalf: calc($gridGap / 2);
+
+.product-form {
+  display: grid;
+  grid-template-columns: calc(30% - $gridGapHalf) calc(70% - $gridGapHalf);
+  grid-gap: $gridGap;
+  margin-bottom: spacing(5);
+
+  &__header {
+    background-color: var(--new-product-form-head-bg-color, $light-alpha);
+    padding: spacing(5) spacing(8);
+
+    &-main {
+      display: flex;
+      gap: spacing(2);
+      padding: 11.5px spacing(8) !important; // to overwrite mui styles
+      align-items: center;
+    }
+  }
+
+  &__body {
+    padding: spacing(8);
+  }
+
+  &__container {
+    box-shadow: var(--new-product-form-container-shadow, $shadow-black);
+    background-color: var(--new-product-form-container-bg-color, $white);
+  }
+
+  &__main-info-container {
+    flex: 3 0 auto;
+  }
+
+  &__image-preview {
+    border: 1px solid var(--spa-accordion-container-border-color, $gray-light);
+    border-radius: 3px;
```

Comment:
same for that. Can we use `spacing(1)` here?

Reply:
Thank you, it can be used there. I used `spacing(1)`